templates/routing-informs/part2.sh

```diff
@@ -107,14 +107,15 @@ Now let's run an experiment comparing the results of our updated version to that
 production instance. An experiment runs two versions (represented by instances) against the same
 set of inputs, and compares the results.
 The following command will be run to compare the results of production and staging:
-nextmv experiment batch start \\
-      --experiment-id "compare-with-service-times" \\
-      --app-id $APPID \\
-			--instance-ids "prod,staging" \\
+  nextmv experiment batch start \\
+    --app-id $APPID \\
+	--experiment-id "compare-with-service-times" \\
+    --instance-ids "prod,staging" \\
 			--input-set-id denver-sample \\
-      --name "Compare v1 prod to v2 staging" \\
-			--description "Comparison with service times"
-Press return to create the staging instance, with our updated version assigned.
+    --name "Compare v1 prod to v2 staging" \\
+    --description "Comparison with service times"
+Press return to create an experiment on the production and staging instances
```

Comment:
```suggestion
Press return to create an experiment comparing the production and staging instances
```

I'm not sure if the above is more clear or not but that is how I think about it.